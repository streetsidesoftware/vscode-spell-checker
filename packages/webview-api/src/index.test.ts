import { describe, expect, test } from 'vitest';

import * as api from './index';

describe('api', () => {
    test('api', () => {
        expect(Object.keys(api).sort()).toMatchSnapshot();
    });
});
