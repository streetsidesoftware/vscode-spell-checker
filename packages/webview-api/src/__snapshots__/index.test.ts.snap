// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`api > api 1`] = `
[
  "createClientSideSpellInfoWebviewApi",
  "createServerSideSpellInfoWebviewApi",
  "supportedViewsByName",
]
`;
